format/spec.md

```diff
@@ -121,9 +121,9 @@ Tables do not require random-access writes. Once written, data and metadata file
 Tables do not require rename, except for tables that use atomic rename to implement the commit operation for new metadata files.
 
 
-# Specification
+## Specification
```

Comment:
I think that this is the only problem. It appears that if there are multiple H1 sections, only the first one shows up in TOC. For the other changes, we do want to avoid over-nesting, which was originally added a few website iterations ago to get things to _stop_ showing up in the TOC.

I think for at least a few of these, we can just change Specification and not other sections. For example, `Terms` is already H3, so there's no need to make it H4 because there is no intermediate H3 level above it. Everything before Schemas and Data Types falls in that category.

Reply:
ok, I just changed spec to H2 and reverted other changes. Rendered site LGTM. 

Updated the PR.